nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -102,6 +103,45 @@ scenarioTypes {
       classPath: [${modelDir}"/defaultModel.jar", ${liteComponentsDir}"/liteRequestResponse.jar", ${liteComponentsDir}"/liteBase.jar"]
     }
   }
+  "lite-streaming-k8s": {
+    deploymentConfig {
+      type: "lite-streaming-k8s"
+      configExecutionOverrides {
+        metrics {
+          environment: ${environment},
+          influx {
+            url: "http://influxdb:8086/write",
+            database: "esp",
+          }
+        }
+        modelConfig {
+          kafka {
+            kafkaAddress: "kafka:9092"
+            kafkaProperties {
+              "schema.registry.url": "http://schemaregistry:8081"
+            }
+          }
+        }
+      }
+    }
+    modelConfig {
+      classPath: [${modelDir}/"defaultModel.jar", ${componentsDir}"/lite/liteBase.jar", ${componentsDir}"/lite/liteKafka.jar", ${componentsDir}"/openapi.jar", ${componentsDir}"/sql.jar"]
+      categories: ["LiteStreamingK8s"]
+
+      components.kafka.enabled: true
```

Comment:
why it should be specified? kafka lite isn't auto loaded?

Reply:
true, just copied it unnecessarily